lib/vttcue.js

```diff
@@ -224,10 +228,11 @@ function VTTCue(startTime, endTime, text) {
       set: function(value) {
         var setting = findAlignSetting(value);
         if (!setting) {
-          throw new SyntaxError("positionAlign: An invalid or illegal alignment string was specified.");
+          console.warn("positionAlign: An invalid or illegal string was specified.");
```

Comment:
Extremely minor/nitpicky, but the two warnings logged have different casing on the word `an`. Change if you like.